src/pages/Sidebar.js

```diff
@@ -0,0 +1,156 @@
+import { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { FaSwatchbook } from 'react-icons/fa';
+import {
+  AiFillDatabase,
+  AiFillDelete,
+  AiFillHome,
+  AiFillBank,
+} from 'react-icons/ai';
+import { BiLogOutCircle } from 'react-icons/bi';
+import { HiMenu } from 'react-icons/hi';
+import { NavLink } from 'react-router-dom';
+import { RiCloseLine } from 'react-icons/ri';
+import { getCurrentUser, userLogout } from '../redux/users/usersSlice';
+import '../styles/sidebar.css';
+
+const Sidebar = () => {
+  const currentUser = useSelector((state) => state.user.currentUser);
```

Comment:
Can be written liek so: 
`const { currentUser } = useSelector((state) => state.user);`